README.md

```diff
@@ -1,5 +1,9 @@
 # Authenticating with the TMC
 
+```
+❗tmc-auth distribution has moved! Find us on CloudSmith.
+```
+
```

Comment:
We should either add more information or remove this statement from here. There is no other reference to Bintray or distribution on this page. Anyone reading this would wonder why this is important and what, if anything, should they do?

Reply:
I was hoping you might suggest better wording. Are you a writer Jenish? I couldn't think of what to say.

Reply:
We are all writers on this team 😆 . Here is my suggestion.

```suggestion
> :warning: **Our artifacts have moved from Bintray to Cloudsmith.**
> If you are using version `3.0.5-beta` or an older version, you will need to replace Bintray URL `https://autonomic.bintray.com/au-tmc-oss` to Cloudsmith URL `https://dl.cloudsmith.io/public/autonomic/au-tmc-oss/maven/`.
> - If you are using Maven, these URLs can be found in `settings.xml`,  or `pom.xml`.
> - If you are using Gradle, these URLs can be found in`build.gradle`,  or `build.gradle.kts`.
```

Reply:
I like where Jenish has started. What do you think about something like this?


---
|:speaking_head: ANNOUNCEMENT: Autonomic's artifacts are moving from Bintray to [Cloudsmith](https://cloudsmith.com/)|
|---|

If you are using `tmc-auth 3.0.5-beta` or earlier, you will need to replace:
  -  Your Bintray URL `https://autonomic.bintray.com/au-tmc-oss`
 -  With a Cloudsmith URL `https://dl.cloudsmith.io/public/autonomic/au-tmc-oss/maven/`

If you are using Maven, these URLs can be found in your `settings.xml`,  or `pom.xml`.

If you are using Gradle, these URLs can be found in your `init.gradle`, `build.gradle`,  or `build.gradle.kts`.

---

Code:
```
---
|:speaking_head: ANNOUNCEMENT: Autonomic's artifacts are moving from Bintray to [Cloudsmith](https://cloudsmith.com/)|
|---|

If you are using `tmc-auth 3.0.5-beta` or earlier, you will need to replace:
  -  Your Bintray URL `https://autonomic.bintray.com/au-tmc-oss`
 -  With a Cloudsmith URL `https://dl.cloudsmith.io/public/autonomic/au-tmc-oss/maven/`

If you are using Maven, these URLs can be found in your `settings.xml`,  or `pom.xml`.

If you are using Gradle, these URLs can be found in your `init.gradle`, `build.gradle`,  or `build.gradle.kts`.

---
```

Reply:
Given that we will only make this change public AFTER artifacts have moved to Cloudsmith, I think we should use `have moved` rather than `are moving` in the announcement line.

Reply:
What about the alligators info box?
```
> ℹ️ info box with alligators?
```

Reply:
> 💔  We had our heart broken by JFrog. Thus Autonomic's artifacts are moving to from Bintray to Cloudsmith

Reply:
> Given that we will only make this change public AFTER artifacts have moved to Cloudsmith, I think we should use `have moved` rather than `are moving` in the announcement line.

I agree